### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by mishkanemes** [alan-turing-institute/learning-at-the-turing#28]
- [ ] Agree where this is applicable eg GitHub, Slack, Moodle, live events etc
- [ ] Review existing CoC 
- [ ] Publish it and link in relevant comms

**Comment by mishkanemes** [comment]
Current version [here](https://thealanturininstitute.sharepoint.com/:w:/s/SkillsTeam/EbjHrEPFh8dPjFX2MlzNU20Ba_ndqoOTJ5kf7JtCFFnWnQ?e=Ygx38B)

**Comment by AyeshaDunk** [comment]
I have my own CoC, adapted from the Turing Way, for Slack and Educators' Prog. Is this what this is adapted from?

**Comment by snietopski** [comment]
@AyeshaDunk yes this was also adapted from TW - made some edits to make it specific to Moodle but maybe we need it to be more general for all Skills activities?

**Comment by AyeshaDunk** [comment]
I think it would make more sense to have an overall CoC for different platforms, rather than all Skills activities. The only issue here would be that the 'designated' contact would be different for different projects, unless one person in the team would need to be the designated contact for all CoC?
